keyboards/idobao/id67/id67.c

```diff
@@ -56,6 +56,6 @@ led_config_t g_led_config = { {
 __attribute__ ((weak))
 void rgb_matrix_indicators_user(void) {
```

Comment:
this entire function should be removed
1. caps lock state should not be coded into the core file.  also #FFFFFF is really bright !  This value should be based on current brightness values
2. value would have been 36.  idx-2-phys & led-idx are reversed, but not the kmtrx-2-led-idx in `g_led_config`